timers.js

```diff
@@ -1,4 +1,29 @@
+  var resetButton = document.querySelector("#reset");
+  var startButton = document.querySelector("#start");
+  var pauseButton = document.querySelector("#pause");
+  var stopWatch = document.querySelector("#timer");
+  var clicks = 0;
+  var seconds = 0;
 
+  function updateTime() {
+    stopWatch.innerHTML = ("Time Elapsed: " + seconds++);
+  };
 
 
+  startButton.addEventListener("click", function(){
+    clicks++;
```

Comment:
This approach works, but requires a bit of thought to understand what it's doing.

A clearer way may be to check the presence of the timerId variable... if it's 'truthy', then no need to set the interval.